templates/AddNewLibrary.ps1

```diff
@@ -67,8 +67,13 @@ $Replacements = @{
 Copy-Item -Recurse -Path $TemplateDirectories -Destination $Src
 $SrcDirectories |% { Replace-Placeholders -LibraryName $LibraryName -Replacements $Replacements -Path $_ }
 
-Write-Output "Great. Your two new projects have been created."
-Write-Output "Please add these new projects to your solution file:"
+Write-Output "Great. Your new projects have been created."
+Write-Output "Now you need to create the $LibraryName.exports.txt file."
+Write-Output "Please use the `"templates\CreateNativeTxtFiles.ps1`" Powershell cmdlet to generate the mentioned file."
+Write-Output "    templates\CreateNativeTxtFiles.ps1 `"`$env:windir\System32\$LibraryName.dll`""
+Write-Output "Add the generated $LibraryName.exports.txt to either the $LibraryName.Desktop or $LibraryName project as appropriate."
+Write-Output "Make sure $LibraryName.exports.txt gets copied as a build output."
```

Comment:
> Which is the "right place"

Good question. For almost every project so far, where we have both portable and desktop, desktop is always the right choice (since it is a superset of portable). For the very few projects that only have portable varieties, it would be otherwise. Given the strong bias toward having a desktop project, I suggest we script it to go there.

> If we can automate this process should the CreateNativeTxtFiles.ps1 script be included as a function inside the AddNewLibrary.ps1 script?

No. Keep it separate but invoke it from there. The reason for it is this: users may want to invoke the script by itself, for example if it fails the first time, or if some update to Windows adds more exported functions.